### PR TITLE
Filter added for informTouristAndFreight and tests

### PR DESCRIPTION
**Pull request by adam-thorpe**
- If a target has the informTouristAndFreight flag it will now also be returned as part of the filter if either RORO Tourist or RORO Accompanied Freight is selected.
- If it does not have the flag, or if neither RORO Tourist or RORO Accompanied Freight is selected, then it will not be returned
- I used XOR logic for this because if both are selected then all possible targets will already be returned
- Added tests

### CODE REVIEW

File: src/main/java/uk/gov/homeoffice/cop/workflow/targeting/domain/MovementMode.java [hunk 1]
```diff
@@ -14,4 +16,11 @@ public static MovementMode fromString(String value) {
         return MovementMode.valueOf(value.toUpperCase().replaceAll("-", "_"));
     }
 
+    public static boolean shouldInformTouristAndFreightCollection(Collection<MovementMode> input) {
+        if (input.contains(RORO_TOURIST) ^ input.contains(RORO_ACCOMPANIED_FREIGHT)) {
```
Comment (michaelruocco) [inline src/main/java/uk/gov/homeoffice/cop/workflow/targeting/domain/MovementMode.java:20]: dont often see this used, from memory is it basically one or the other? do you need the if or could it just do?

`return input.contains(RORO_TOURIST) ^ input.contains(RORO_ACCOMPANIED_FREIGHT;`

**Review by michaelruocco — APPROVED** [review]
LGTM

File: src/main/java/uk/gov/homeoffice/cop/workflow/targeting/domain/MovementMode.java (hunk 1 above)
Comment (adam-thorpe) [reply src/main/java/uk/gov/homeoffice/cop/workflow/targeting/domain/MovementMode.java:20]: Oh yeah I suppose probably not, I'll quickly tweak it

**Review by michaelruocco — APPROVED** [review]
LGTM

**Pull request merged**